godot-resources/mods/RaceAgainstMod/src/environments/raceagainst/Coordinates.gd

```diff
@@ -0,0 +1,17 @@
+extends Node
+
+
+# Declare member variables here. Examples:
+# var a = 2
+# var b = "text"
+
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	pass # Replace with function body.
+
+
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+func _process(delta):
+	print()
+	pass
```

Comment:
I realize this was for testing purposes. But should probably be removed on master.

Reply:
Never mind, realized it was to print coordinates.